blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/RollupRunnable.java

```diff
@@ -65,7 +69,38 @@ public RollupRunnable(RollupExecutionContext executionContext, SingleRollupReadC
         this.rollupBatchWriter = rollupBatchWriter;
         startWait = System.currentTimeMillis();
     }
-    
+
+    private static DiscoveryIO loadDiscoveryModule() {
```

Comment:
Loading the discovery module, seems to be a bit out of place in `RollupRunnable`...can you look into if we can refactor `RollupEvent` and  REE so that we do a check if there are current subscribers to the rollup event and get units from MetadataCache, (yes, without concerning about ES in that case) there?
Let me know if you want some more explanation.